### PR TITLE
Add tutorial

### PR DESCRIPTION
**Pull request by clementzach**
Adding an initial notebook to the tutorial

### CODE REVIEW

File: tutorials/.ipynb_checkpoints/forest_usage-checkpoint.ipynb [hunk 1]
```diff
@@ -0,0 +1,579 @@
+{
```
Comment (hackdna) [inline tutorials/.ipynb_checkpoints/forest_usage-checkpoint.ipynb:1]: Do you actually need to include this file? Shouldn't `forest_usage.ipynb` be sufficient?

File: tutorials/forest_usage.ipynb [hunk 2]
```diff
@@ -0,0 +1,579 @@
...
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "3.10.1\n",
+      "/Library/Frameworks/Python.framework/Versions/3.10/bin/python3.10\n"
```
Comment (hackdna) [inline tutorials/forest_usage.ipynb:37]: Python 3.8 should be used here

**Review by hackdna — CHANGES_REQUESTED** [review]
Thank you! Just a couple of issues. Also, since you are running this notebook using Forest code from the main branch (and you want this as part of the current release), you'll need to edit this PR to merge into main instead of develop.

File: tutorials/forest_usage.ipynb (hunk 2 above)
Comment (clementzach) [reply tutorials/forest_usage.ipynb]: Those are both done

File: tutorials/.ipynb_checkpoints/forest_usage-checkpoint.ipynb (hunk 1 above)
Comment (clementzach) [reply tutorials/.ipynb_checkpoints/forest_usage-checkpoint.ipynb]: Yeah, the ipynb document should be sufficient. Thanks for catching that!

File: .gitignore [hunk 3]
```diff
@@ -11,5 +11,7 @@ forest.egg-info/
 tutorials/data
 tutorials/comm_output
 tutorials/gps_output
+tutorials/gps_summary.csv
 
-
+#Remove checkpoints data because we don't need to double-track files
+tutorials/.ipynb_checkpoints/forest_usage-checkpoint.ipynb
```
Comment (hackdna) [inline .gitignore:17]: Should the whole directory be excluded?
`tutorials/.ipynb_checkpoints/`

**Review by hackdna — CHANGES_REQUESTED** [review]
Thanks! Since you are running this notebook using Forest code from the main branch (and you want this as part of the current release), you'll need to edit this PR to merge into main instead of develop.

**Comment by clementzach** [comment]
I changed it to merge into main.

Thanks for working with me on this! Sorry I'm still learning a lot about using GitHub.

File: .gitignore [hunk 4]
```diff
@@ -6,3 +6,12 @@ __pycache__/
...
 forest.egg-info/
+
+#We don't want to add data to github
+tutorials/data
+tutorials/comm_output
+tutorials/gps_output
+tutorials/gps_summary.csv
+
+#Remove checkpoints data because we don't need to double-track files
+tutorials/.ipynb_checkpoints
```
Comment (hackdna) [inline .gitignore:17]: This is a directory, so should have a trailing slash: `tutorials/.ipynb_checkpoints/`
https://www.atlassian.com/git/tutorials/saving-changes/gitignore#git-ignore-patterns

**Review by hackdna — CHANGES_REQUESTED** [review]
Thanks and no worries! This will be more straightforward in the future when all new code will go into the develop branch.

File: .gitignore (hunk 4 above)
Comment (hackdna) [reply .gitignore:17]: Actually, to be precise it should be `/tutorials/.ipynb_checkpoints/` for this directory. The same applies for the lines above.

**Review by hackdna — APPROVED** [review]
Thank you!

**Pull request merged**